src/test/java/api/requests/RequestScheduledNoticesTests.java

```diff
@@ -81,10 +81,11 @@ public void requestExpirationUponAtNoticeShouldBeScheduledWhenCreatedRequestIsSe
       .withName("request policy")
       .withRequestNotices(Collections.singletonList(requestNotice));
 
-    useLoanPolicyAsFallback(
+    useFallbackPolicies(
```

Comment:
This is repeated 5 times and should be extracted into something like `useWithPaging(NoticePolicyBuilder noticePolicyBuilder)`

Reply:
Done.